### PR TITLE
RichCommand: InstallFont

### PR DESCRIPTION
**Pull request by mafra99**
**Resolved / Related Issues**
- [ ] Were these changes approved in an issue or discussion with the project maintainers? In order to prevent extra work, feature requests and changes to the codebase must be approved before the pull request will be reviewed. This prevents extra work for the contributors and maintainers.
   Relates #11481 

**Validation**
How did you test these changes?
- [x] Did you build the app and test your changes?
- [ ] Did you check for accessibility? You can use Accessibility Insights for this.
- [ ] Did you remove any strings from the en-us resource file?
   - [ ] Did you search the solution to see if the string is still being used? 
- [ ] Did you implement any design changes to an existing feature?
   - [ ] Was this change approved?
- [x] Are there any other steps that were used to validate these changes?
   1. Install a font from the toolbar
   2. Install a font from the menu flyout
   3. Install button is hidden if no item is selected or in recycle bin

**Screenshots (optional)**
Add screenshots here.

### CODE REVIEW

File: src/Files.App/Shell/Win32API.cs [hunk 1]
```diff
@@ -805,12 +805,19 @@ public static async Task<bool> InstallInf(string filePath)
 			}
 		}
 
-		public static void InstallFont(string fontFilePath)
+		public static void InstallFont(string fontFilePath, bool forAllUsers)
```
Comment (yaira2) [inline src/Files.App/Shell/Win32API.cs:808]: Are these changes related to the rich command? If not, we should limit the scope of this PR.

File: src/Files.App/Actions/Content/InstallFontAction.cs
[the remark this answers is not in the document]
Comment (yaira2) [reply src/Files.App/Actions/Content/InstallFontAction.cs:1]: I think that's fair

**Review by ferrariofilippo — CHANGES_REQUESTED** [review]
Overall lit ooks good, just one little thing with the style

**Review by QuaintMako — APPROVED** [review]
LGTM

**Pull request merged**